tests/Doctrine/Tests/DBAL/Schema/PostgreSqlListTableFunctionsTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Schema;
+
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Platforms\PostgreSqlPlatform;
+use Doctrine\Tests\DbalFunctionalTestCase;
+use function extension_loaded;
+use function in_array;
+
+class PostgreSqlListTableFunctionsTest extends DbalFunctionalTestCase
+{
+    protected function setUp() : void
+    {
+        if (! extension_loaded('pdo_pgsql')) {
+            $this->markTestSkipped('pdo_pgsql is not loaded.');
+
+            return;
+        }
+
+        parent::setUp();
+
+        if (! in_array('pdo_pgsql', DriverManager::getAvailableDrivers())) {
+            $this->markTestSkipped('PostgreSQL driver not available');
+
+            return;
+        }
+
+        if (! $this->connection->getDatabasePlatform() instanceof PostgreSqlPlatform) {
+            $this->markTestSkipped('PostgreSQL Only test.');
+
+            return;
+        }
+
+        $this->connection->executeQuery('CREATE SCHEMA test_schema1;');
+        $this->connection->executeQuery('CREATE SCHEMA test_schema2;');
+        $this->connection->executeQuery('set search_path = "test_schema1", "test_schema2";');
+        $this->connection->executeQuery('CREATE TABLE test_schema1.test_foreign1 ( id integer PRIMARY KEY );');
+        $this->connection->executeQuery('CREATE TABLE test_schema1.test_table ( test_column1 varchar(5), test_foreign1_id integer constraint fk_test_foreign1 references test_schema1.test_foreign1 (id) );');
+        $this->connection->executeQuery('CREATE INDEX idx_test_column1 ON test_schema1.test_table (test_column1);');
+        $this->connection->executeQuery('CREATE TABLE test_schema2.test_foreign2 ( id integer PRIMARY KEY );');
+        $this->connection->executeQuery('CREATE TABLE test_schema2.test_table ( test_column2 varchar(5), test_foreign2_id integer constraint fk_test_foreign2 references test_schema2.test_foreign2 (id) );');
+        $this->connection->executeQuery('CREATE INDEX idx_test_column2 ON test_schema2.test_table (test_column2);');
+    }
+
+    /**
+     * @group legacy
+     * @expectedDeprecation Providing a table name without a schema prefix, i.e. 'some_schema.some_table', (%s given) is deprecated and will cause an error in Doctrine 3.0
```

Comment:
Does this actually work? I would expect https://github.com/doctrine/dbal/pull/3643 to disable this. If yes, we forgot to remove other occurrences of `@expectedDeprecation` in that PR.

Reply:
Before I rebased to `doctrine:master` I had to have that there for the test to pass. It appears that this test will now pass without having it there. Do you want me to take this out?

Reply:
Yeah because it's a bit misleading now: we are saying that something is asserted, and it actually isn't:
- false sense of security
- wtf effect if the actual message is changed in a subsequent PR for some reason and that message isn't, and the tests still pass

To be clear, we're talking about line 50, line 49 has to stay.

Reply:
Ok, I've removed it. 
BTW there are three other tests that still have the `@expectedDeprecation` notation.
`PDOStatementTest.php testPDOSpecificModeIsAccepted (line 33)`
`ColumnTest.php testSettingUnknownOptionIsStillSupported (line 67)`
`ColumnTest.php testOptionsShouldNotBeIgnored (line 78)`

Reply:
I'm going to handle these in a separate PR, thanks.

Reply:
See #3719